warehouse/query-core/src/main/java/datawave/query/jexl/visitors/BoundedRangeIndexExpansionVisitor.java

```diff
@@ -104,7 +104,39 @@ protected void rebuildFutureJexlNode(FutureJexlNode futureJexlNode) {
         JexlNode currentNode = futureJexlNode.getOrigNode();
         IndexLookupMap fieldsToTerms = futureJexlNode.getLookup().lookup();
 
+        if (log.isDebugEnabled()) {
+            logResult(currentNode, fieldsToTerms);
+        }
+
         futureJexlNode.setRebuiltNode(JexlNodeFactory.createNodeTreeFromFieldsToValues(JexlNodeFactory.ContainerType.OR_NODE, false, currentNode, fieldsToTerms,
                         expandFields, expandValues, futureJexlNode.isKeepOriginalNode()));
     }
+
+    /**
+     * Log the result of index expansion for the provided term.
+     *
+     * @param node
+     *            the term
+     * @param lookupMap
+     *            the result of the lookup
+     */
+    protected void logResult(JexlNode node, IndexLookupMap lookupMap) {
+        String field = JexlASTHelper.getIdentifier(node);
+
+        if (lookupMap == null) {
+            log.debug("Lookup map was null for term [{}]", JexlStringBuildingVisitor.buildQuery(node));
+            return;
+        }
+
+        if (lookupMap.containsKey(field)) {
```

Comment:
Love it.  My only though is since the logResult methods in the visitors are extremely similar, this could probably be bumped to a base Visitor and supply the strings that are different, i.e., range, regex, or field expansion

Reply:
easy enough